src/states/node.rs

```diff
@@ -2021,6 +2114,27 @@ impl Node {
         self.merge_if_necessary();
     }
 
+    pub fn section_list_signatures(&self,
+                                   prefix: Prefix<XorName>)
+                                   -> BTreeMap<PublicId, sign::Signature> {
+        if let Some(&(ref section_list, ref signatures)) =
+            self.section_list_sigs.get_signatures(prefix) {
+            let data = if let Ok(data) = serialisation::serialise(section_list) {
+                data
+            } else {
+                return Default::default();
```

Comment:
We should log the error here (or alternatively return a `Result`, not sure which is cleaner).